src/index.js

```diff
@@ -1,3 +1,46 @@
 import './index.scss';
 
-console.log('Hello World!');
+const gameId = 'JANudAkE59i9uvoF85MJ';
+
+const user = document.querySelector('.form__element--name');
+const score = document.querySelector('.form__element--score');
+const formSubmitButton = document.querySelector('.form__button');
+const refreshButton = document.querySelector('.main__button');
+const ul = document.querySelector('.ul');
+
+// POST data to API
+formSubmitButton.addEventListener('click', () => {
+  const options = {
+    method: 'POST',
+    body: JSON.stringify({ user: user.value, score: score.value }),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  };
+
+  fetch(
+    `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`,
+    options,
+  )
+    .then((response) => response.json())
+    .then((data) => console.log(data));
+});
+
+// GET data from API
+const fetchData = async () => {
```

Comment:
- [ ] [OPTIONAL] After clicking the Refresh button the list of scores duplicates kindly make sure to clear the old data before displaying the new ones

![Screenshot_20230413_141852](https://user-images.githubusercontent.com/25227022/231788921-1ab13e23-4130-4607-8b3d-d83e40dde35b.png)

Reply:
Hello @Ismailco 👋,

Thanks for finding that bug. It's already fixed.

Regards!!!!!!